packages/git-resolver/src/parsePref.ts

```diff
@@ -30,6 +30,9 @@ const gitProtocols = new Set([
 ])
 
 export default async function parsePref (pref: string): Promise<HostedPackageSpec | null> {
+  if (pref.startsWith('https://github.com/') || pref.startsWith('http://github.com/')) {
+    pref = pref.replace(/^https?:\/\/github\.com\//, '')
+  }
```

Comment:
Should we also support for gitlab and bitbucket URLs?

Reply:
If possible, definitely.